src/Parsers/ParserRenameQuery.cpp

```diff
@@ -67,6 +68,7 @@ bool ParserRenameQuery::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
         ASTPtr from_db;
         ASTPtr to_db;
         ParserIdentifier db_name_p;
+        bool if_exists = s_if_exists.ignore(pos, expected);
```

Comment:
Also need to update `ASTRenameQuery::formatQueryImpl(...)` to make formatter consistent with parser.

Reply:
done.